bucket/emule-community.json

```diff
@@ -0,0 +1,24 @@
+{
+    "homepage": "https://www.emule-project.net/",
+    "description": "One of the biggest and most reliable peer-to-peer file sharing clients around the world.",
+    "version": "0.51c",
+    "license": "GPLv2",
+    "url": "https://files.emule-project.net/eMule0.51c.zip",
+    "hash": "sha512:dd93edad816e17ee85ffda3f7bd228ea3e4f06e53402415dce4e6e8b934d2d7606d77ab40eed264e7713fc24ecf37c9f2904d49da402b39270bc2de380f903cc",
```

Comment:
What should I do? 
This line is copied from other json file and I replace it with the corresponding sha512 hash.

Reply:
Run checkver for both files

Reply:
version regexs have been updated. But what should I do with the 512 hash?

Reply:
As I said. Run checkver.

`.\bin\checkver.ps1 emul* -f`

Reply:
```
Downloading eMule0.51c.zip to compute hashes!
Loading eMule0.51c.zip from cache
Computed hash: 601affcde7ce890c217d3779bfda89e49653389f61bf9919e2e332ef8796495e
Writing updated emule-community manifest
emule: 0.50a (scoop version is 0.50a)
Forcing autoupdate!
Autoupdating emule
Found: sha1:8403ff232b1a9d8a7a9ed45045f5061776919c73 using Sourceforge Mode
Writing updated emule manifest
```
Amazing. How can checkver know it can get sha1 hash from sourceforge? Where is the hash of emule community from? Does checkver calculate it by itself?

Reply:
I thought checkver is only used for user to check whether the regex is right or not. I didn't know it could fix hashes.

Reply:
https://github.com/lukesampson/scoop/blob/1ac8ac0c6506f8012e18ba143f315abcc95ef836/lib/autoupdate.ps1#L223-L254